shell/common/gin_helper/dictionary.h

```diff
@@ -134,6 +134,8 @@ class Dictionary : public gin::Dictionary {
 
   bool IsEmpty() const { return isolate() == nullptr || GetHandle().IsEmpty(); }
 
+  v8::Isolate* GetIsolate() const { return isolate(); }
```

Comment:
I think users can just use `options.isolate()`? All `gin::Dictionary` members should be visible to users.